hoot-core/src/main/cpp/hoot/core/io/BulkInsert.cpp

```diff
@@ -22,6 +22,6 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
I think we are going to have an issue where files committed with the new script are not going to match with files committed with the old script for 2017. The more the old one is executed by more people, the less full copyright history we're going to have as people clone the hoot repo. Last night, during testing, I noted 642 files needing copyright header updating. After beginning with a *fresh* set of files (I deleted my hoot directory and re-cloned), I noticed that only 76 files needed copyright header updating. This may not be that big of a deal but I feel this point needs to be raised for SA purposes.